### PR TITLE
Add commands in README file

### PR DESCRIPTION
**Pull request by velychkovskyi**
## What Are We Doing Here?

Added commands in README file

## How to Test and Verify

Run communds and check the result

### Level

`Low`

### Required Testing

`Smoke` or `Manual`

### Risk Summary

This steps already are automated

## Code Review Checklist

- [ ] Testing instructions are included. If not, make sure you leave us a line or two for the reason.

- [ ] Well documented, including updates to any necessary markdown files. When
we inevitably come back to this code it will only take hours to figure out, not
days.

- [ ] Make sure PR has only the changes required for the Jira issues.

- [ ] Have you make sure to follow the agreements in our process document?

- [ ] Have someone from different region than you review your changes. If not, make sure to leave a comment on why it is not required

- [ ] If it's applicable, document deployment plan for the changes.

### CODE REVIEW

**Review by vecinomio — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -5,18 +5,51 @@ The main goal is not to create complex functionality, but configure optimal CI/C
...
+    $ sudo apt update 
+    $ sudo apt install openjdk-8-jdk
+    $ java -version
+
 Install Tomcat 8+ to run compiled WAR file.
 
-## Manual deployment
+For Ubuntu or Debian would run:
+
+    $ sudo groupadd tomcat
```
Comment (mike-unitskyi) [inline README.md:18]: it's worth to describe procedure with docker, as you have already had Dockerfile

Comment (velychkovskyi) [reply README.md:18]: Ya, added such section

File: README.md [hunk 2]
```diff
@@ -5,20 +5,71 @@ The main goal is not to create complex functionality, but configure optimal CI/C
...
 The script compiles the project's classes, packs the necessary files into a web archive
 
 - Paste WAR file to Tomcat server.
+
+        $ sudo rm -rf /opt/tomcat/webapps/ROOT/
+        $ cd eb-tomcat-snakes && cp ROOT.war /opt/tomcat/webapps/
+
 Open [localhost:8080](http://localhost:8080/) in a web browser to check the running application.
 
+## Сontainerized deployment
```
Comment (mike-unitskyi) [inline README.md]: typo, `deployment` -> `development`

File: README.md [hunk 3]
```diff
@@ -5,20 +5,71 @@ The main goal is not to create complex functionality, but configure optimal CI/C
...
+We can use containerized tool - Docker to run the application.
```
Comment (mike-unitskyi) [inline README.md]: add instruction how to install docker.

File: README.md [hunk 4]
```diff
@@ -5,20 +5,71 @@ The main goal is not to create complex functionality, but configure optimal CI/C
...
+Repository already have Dockerfile which contain strict commands for container building.
+
+### Build docker image with application:
+Navigate to folder that contain Dockerfile and build:
+
+    $ docker build -t snakes-web-application .
```
Comment (mike-unitskyi) [inline README.md:75]: should app be built before docker execution?

File: README.md (hunk 2 above)
Comment (velychkovskyi) [reply README.md]: As for me it is coupled things (here), so when I typed deployment I meant  development as a part.
But ok, corrected

File: README.md (hunk 3 above)
Comment (velychkovskyi) [reply README.md]: Added

File: README.md (hunk 4 above)
Comment (velychkovskyi) [reply README.md:75]: Yes, true. 
Wrote such step.

**Review by mike-unitskyi — APPROVED** [review]

**Pull request merged**